engine/gfx/fade.asm

```diff
@@ -190,10 +190,10 @@ FadePalettesStep:
 	ld a, [wPalFadeMode]
 	and PALFADE_WHICH
 	ld hl, wBGPals2
-	ld d, 4 * 16 ; colors, palettes
+	ld d, PAL_COLORS * 16 ; colors * number of palettes
```

Comment:
I think the comment can just be removed, the constant usage is clear enough. Same for the one below. If you think it needs clarification, "`; bg + ob pals`" and below "`; ob pals only`" could be more helpful.